packages/serverpod_client/lib/src/serverpod_client_exception.dart

```diff
@@ -14,3 +16,39 @@ class ServerpodClientException implements Exception {
     return ('ServerpodClientException: ${message.trim()}, statusCode = $statusCode');
   }
 }
+
+/// Thrown if the client created a malformed or invalid request
+/// to the server.
+class ServerpodClientBadRequest extends ServerpodClientException {
+  /// Creates a Bad Request Exception
+  ServerpodClientBadRequest() : super('Bad request', HttpStatus.badRequest);
+}
+
+/// Thrown if the client is not authorized to perform the request.
```

Comment:
Suggestion: Rephrase to `Thrown if the client fails to authenticate and is therefore not authorized to perform the request`.

The current phrasing makes me confuse it with 403: Forbidden.